content/docs/user-guide/project-config/_index.md

```diff
@@ -32,16 +33,49 @@ To change any of the properties in this file, you can either edit the file manua
-| **icon_path** | The path and filename of the project icon. Project Manager uses this icon. The file must be located in the project root directory, and the filename must be `preview.png`. <br><br> WARNING: Do not change the path or filename. | "preview.png" |
-| **engine** | The name of the engine registered to the project. Register engines in your O3DE manifest, located at `<USER>/.o3de/o3de_manifest.json`. | "o3de" |
-| **restricted_name** | (Optional) The name of a restricted folder to associate with the project. | "projects" |
-| **external_subdirectories** | (Optional) The path to one or more directories to include in the project build. You can use any directory with a `CMakeLists.txt` file. When you register Gems to the project using the `register --external-subdirectory-project-path` O3DE CLI command, O3DE adds their paths here. | |
+| Property | Required | Description | Default |
+| --- | --- | --- | --- |
+| **project_name** | **Required** | The name of the project. The `--project-name` O3DE CLI parameter uses this name to identify the project. | "<PROJECT_NAME>" |
+| **display_name** | **Required** | The display name of the project in Project Manager. | "<PROJECT_NAME>" |
+| **engine** | Optional | The name of the engine and optional version specifier this project uses. Register engines in your O3DE manifest, located at `<USER>/.o3de/o3de_manifest.json`. | "o3de" |
+| **engine_api_dependencies** | Optional | A list of engine API dependencies.  If empty, the project is assumed compatible with all versions of any engine APIs. | "" |
+| **external_subdirectories** | Optional | The path to one or more directories to include in the project build. You can use any directory with a `CMakeLists.txt` file. When you register Gems to the project using the `register --external-subdirectory-project-path` O3DE CLI command, O3DE adds their paths here. | [ ] |
+| **canonical_tags** | **Required** | A standard field in O3DE manifests used to identify an O3DE object's type. Examples: "Gem", "Project". Projects should use the "Project" tag. | [ "Project" ] |
+| **compatible_engines** | Optional | A list of engine names and optional version specifiers that this project is known to be compatible with: i.e. `o3de>=2.0.0`, `o3de-sdk==1.2.0`, `o3de-custom` etc. If empty, the project is assumed compatible with all engines if they meet all the requirements in the `engine_api_dependencies` and `gem_names` fields. | [ ] |
+| **gem_names** | Optional | A list of gem names and optional version specifiers that this project uses, including the Gem included in the project: i.e. `Atom`, `PopcornFX==1.2.0` etc. | [ "<PROJECT_NAME>" ] |
+| **icon_path** | Optional | The path and filename of the project icon. Project Manager uses this icon. The file must be located in the project root directory, and the filename must be `preview.png`. <br><br> WARNING: Do not change the path or filename. | "preview.png" |
```

Comment:
What is the point of having this key if it cannot be changed?
The WARNING seems wrong to me.

Reply:
it may no longer be valid, will check